src/proto/generation.proto

```diff
@@ -345,6 +345,43 @@ message TransformResample {
   optional TransformMatrix prev_transform = 3; // 3x3 or 4x4 matrix, defaults to identity
   optional float depth_warp = 4;               // depth warp factor, defaults to 1.0
   optional bool export_mask = 5;               // return inpaint mask
+  optional bool do_prefill = 6;                // prefill border areas with values matching the colors around the area
```

Comment:
Instead of the do_prefill we could have BORDER_INPAINT where the inexpensive inpaint is performed. (Since when do_prefill is set the BorderMode is made redundant)

Reply:
Was also thinking about it when added do_prefill, but the fact that do_prefill for camera_pose transform doesn't have anything to do with border_mode stopped me. But I think it should be optimal if I leave do_prefill for CameraPose operation, but hide do_prefill behind BORDER_PREFILL for Resample.